google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java

```diff
@@ -153,4 +191,54 @@ private static String getRequiredProperty(String prop) {
     }
     return value;
   }
+
+  /**
+   * Captures the request attributes "Grpc.TRANSPORT_ATTR_REMOTE_ADDR" when connection is
+   * established and verifies if the remote address is a DirectPath address.
+   * This is only used for DirectPath testing.
+   * {@link ClientCall#getAttributes()}
+   */
+  private ClientInterceptor directPathAddressCheckInterceptor() {
+    return new ClientInterceptor() {
+      private ClientCall<?, ?> capturedCall;
+      @Override
+      public <ReqT, RespT> ClientCall<ReqT, RespT> interceptCall(
+          MethodDescriptor<ReqT, RespT> method, CallOptions callOptions, Channel next) {
+        ClientCall<ReqT, RespT> clientCall = next.newCall(method,callOptions);
+        capturedCall = clientCall;
```

Comment:
I dont think you need capturedCall here, the inner class should be able to access clientCall directly

Reply:
Done.